src/views/extract.html

```
@@ -0,0 +1,29 @@
+    </ul> 
+        <p class="govuk-body">You can review and correct the data that the tool extracts before you export it.</p>
+  <p class="govuk-body">We currently support:</p>
+        <ul class="govuk-list govuk-list--bullet">
+        <li>article 4 directions</li>
+        <li>conservation areas</li>
+        <li>tree preservation orders</li>
+    </ul> 
+  <a href="https://extract.ai.gov.uk/">
+  <span class="govuk-body">Find out more about extract</span>
+  </a>
+  </a>
```

HTMLHint (1.9.2)

[error] Tag must be paired, no start tag: [ </a> ]

(tag-pair)